quetz/main.py

```diff
@@ -1052,7 +1024,12 @@ def handle_package_files(
         if force:
             auth.assert_overwrite_package_version(channel_name, package_name)
 
-        logger.debug(f"adding file '{file.filename}' to channel '{channel_name}'")
+        file.file._file.seek(0, os.SEEK_END)
```

Comment:
I would avoid accessing the _file attribute, it's sort of hack that only works with SpooledTemporaryFile and not a standard file-interface (it's a known issue: https://github.com/python/cpython/pull/3249). You can instead patch the SpooledTemporaryFile as done in CondaInfo: https://github.com/mamba-org/quetz/blob/e552b8afff6df10ceb36c30e5ced47e14de58d44/quetz/condainfo.py#L160-L162

Reply:
Done